content/competitions/_index.html

```diff
@@ -15,9 +15,7 @@ <h1 class="title is-3">UQCS Competitions</h1>
                 We usually run two major competitive events each year: the UQCS CodeJam in Semester 1, and the UQCS Hackathon in Semester 2.
             </p>
 
-            <h4><a href="hackathon-2020">UQCS Hackathon 2020</a> (16th &ndash; 18th October)</h4>
-
-            <h4><a href="codejam-2020">UQCS CodeJam 2020</a> (28th March)</h4>
```

Comment:
Can we keep these but put them in a very obviously-labelled "archive" section? Just for posterity :)